opencti-platform/opencti-graphql/src/modules/organization/organization-domain.ts

```diff
@@ -134,4 +137,33 @@ export const childOrganizationsPaginated = async <T extends BasicStoreCommon> (c
   return listEntitiesThroughRelationsPaginated<T>(context, user, organizationId, RELATION_PART_OF, ENTITY_TYPE_IDENTITY_ORGANIZATION, true, args);
 };
 
+export const organizationDelete = async (context: AuthContext, user: AuthUser, organizationId: string) => {
+  const organization = await findById(context, user, organizationId);
+  if (!organization) {
+    throw AlreadyDeletedError({ organizationId });
+  }
+  const settings = await getEntityFromCache<BasicStoreSettings>(context, SYSTEM_USER, ENTITY_TYPE_SETTINGS);
+  if (organization.id === settings.platform_organization) {
+    throw FunctionalError('Cannot delete the platform organization.');
+  }
+  if (organization.authorized_authorities && organization.authorized_authorities.length > 0) {
+    if (isUserHasCapability(user, SETTINGS_SET_ACCESSES)) {
```

Comment:
If I have BYPASS capability I see the generic error message, should I not see the detailed one?

Reply:
My bad. I can see the detailed message! 😅